protocols/kad/src/behaviour.rs

```diff
@@ -2411,14 +2494,16 @@ pub enum KademliaEvent {
     // is made of multiple requests across multiple remote peers.
     InboundRequest { request: InboundRequest },
 
-    /// An outbound query has produced a result.
-    OutboundQueryCompleted {
+    /// An outbound query has finished.
+    OutboundQueryProgressed {
         /// The ID of the query that finished.
         id: QueryId,
-        /// The result of the query.
+        /// The optional final result of the query.
         result: QueryResult,
         /// Execution statistics from the query.
         stats: QueryStats,
+        /// Indicates which event this is, if therer are multiple responses for a single request.
```

Comment:
```suggestion
        /// Indicates which event this is, if therer are multiple responses for a single query.
```

I think we refer to it as _query_ everywhere else, right?

Reply:
fixed